include/ibamr/STSMassFluxIntegrator.h

```diff
@@ -69,73 +69,72 @@ namespace IBAMR
  * \f$ \nabla \cdot (\rho \mathbf{u} Q)\f$ for the energy equation where \f$ Q = C_\textrm{p} T\f$
  * or \f$ Q = h\f$ based on the form of the energy equation.
  *
- * \note MassIntegrator is an intermediate or a lightweight integrator that is used within one time step to
+ * \note STSMassFluxIntegrator is an intermediate or a lightweight integrator that is used within one time step to
  * get a consistent mass flux \f$ m_\rho = \rho \mathbf{u} \f$ for advecting either momentum \f$ \mathbf{u} \f$
```

Comment:
STSMassFluxIntegrator is an intermediate or a lightweight integrator that is used within one or a single time step (STS) to ...